### PR TITLE
UTS #18 remaining changes

### PR DESCRIPTION
**Pull request by macchiati**

### CODE REVIEW

**Comment by markusicu** [comment]
I know we don't want a formal process in this repo, but for reviewing & feedback it would have been nice to keep this PR open and let me add comments, rather than copy/paste into email. (If there is stuff to say.)

File: tr18/tr18.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
 <meta http-equiv="Content-Language" content="en-us">
-<link rel="stylesheet" type="text/css" href="https://unicode.org/reports/reports-v2.css">
+<link rel="stylesheet" type="text/css" href="http://unicode.org/reports/reports-v2.css">
```
Comment (markusicu) [inline tr18/tr18.html:9]: Isn't this the opposite of what we want?

Comment (macchiati) [reply tr18/tr18.html:9]: I fixed it in a follow-up PR.

**Comment by macchiati** [comment]
Normally I think that's the right process; and we might even think about
requiring reviews.

I wanted to have the result viewable in
https://unicode-org.github.io/unicode-reports/tr18/tr18.html as soon as
possible, however, and for that I needed to merge it.

Mark

**Comment by markusicu** [comment]
> I wanted to have the result viewable in https://unicode-org.github.io/unicode-reports/tr18/tr18.html as soon as possible, however, and for that I needed to merge it.

During review we can slap `https://htmlpreview.github.io/?` in front of the URL to the file in the PR and view it that way.

**Pull request merged**